app-crypt/sops/sops-3.9.2.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit go-module
+
+DESCRIPTION="Simple And Flexible Tool For Managing Secrets"
+HOMEPAGE="https://getsops.io/"
+SRC_URI="https://github.com/getsops/sops/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+SRC_URI+=" https://github.com/Arian-D/sops-deps/releases/download/${PV}/${P}-vendor.tar.xz"
+
+LICENSE="MPL-2.0"
+SLOT="0"
+
+KEYWORDS="amd64 ~arm64"
```

Comment:
You cannot add packages as stable, there is a process for it.

https://devmanual.gentoo.org/keywording/index.html#keywording-new-packages